### PR TITLE
Be more specific about the key ID

### LINKED ISSUE #121: Clearly explain least significant byte in key_id
We should be explicit about what this means:

`"token_key_id" is the least significant byte of the key_id.`

This means that the byte is the last/rightmost byte of the hash, in order. Misinterpretation of this caused some confusion in interop.

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #121.

### CODE REVIEW

**Comment by tfpauly** [comment]
Is this the most exact way to put this? For byte arrays like a hash, I generally don't think of it as being network- or host- byte order. Do we have other docs that refer to stuff like this that we can crib from?

**Comment by chris-wood** [comment]
Nothing comes to mind right now, but I'm happy to look at alternatives.

File: draft-ietf-privacypass-protocol.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ The structure fields are defined as follows:
 
 - "token_type" is a 2-octet integer, which matches the type in the challenge.
 
-- "token_key_id" is the least significant byte of the `key_id`.
+- "token_key_id" is the least significant byte of the `key_id` in network byte order.
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:207]:
```suggestion
- "token_key_id" is the least significant byte of the `key_id` in network byte order (in other words, the last 8 bits of `key_id`).
```

File: draft-ietf-privacypass-protocol.md [hunk 2]
```diff
@@ -380,7 +380,7 @@ The structure fields are defined as follows:
...
```
Comment (tfpauly) [inline draft-ietf-privacypass-protocol.md:383]:
```suggestion
- "token_key_id" is the least significant byte of the `key_id` in network byte order (in other words, the last 8 bits of `key_id`).
```

**Review by tfpauly — COMMENTED** [review]
Here's an attempt at saying this in other words to be extra clear?

File: draft-ietf-privacypass-protocol.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-privacypass-protocol.md:207]: 👍 Great!

**Comment by chris-wood** [comment]
Works for me =)

**Pull request merged**